### ISSUE TITLE
test_sections.py fails locally

### ISSUE THREAD

**Issue by robbievanleeuwen** [robbievanleeuwen/section-properties#154]
On my setup `test_geometry_from_points()` and `test_nested_compound_geometry_from_points()` fails. See below output:

```console
_____________________________________________________________ test_geometry_from_points ______________________________________________________________

    def test_geometry_from_points():
        # Geometry.from_points() tests a shape with exactly one exterior
        # and an arbitrary number of interiors being built from the legacy
        # points, facets, holes, control_points interface of sectionproperties
        exterior = [[-6, 10], [6, 10], [6, -10], [-6, -10]]
        interior1 = [[-4, 8], [4, 8], [4, 4], [-4, 4]]
        interior2 = [[-4, -8], [4, -8], [4, -4], [-4, -4]]
        points = exterior + interior1 + interior2
        facets = [
            [0, 1],
            [1, 2],
            [2, 3],
            [3, 0],
            [4, 5],
            [5, 6],
            [6, 7],
            [7, 4],
            [8, 9],
            [9, 10],
            [10, 11],
            [11, 7],
        ]
        control_points = [[0, 0]]
        holes = [[0, 6], [0, -6]]
        new_geom = Geometry.from_points(points=points, facets=facets, control_points=control_points, holes=holes)
>       assert (
            new_geom.geom.wkt
            == "POLYGON ((6 10, 6 -10, -6 -10, -6 10, 6 10), (-4 4, 4 4, 4 8, -4 8, -4 4), (4 -8, 4 -4, -4 -4, -4 -8, 4 -8))"
        )  # Note, the order of point coordinates can change based on geometry operations, e.g. +/- and how many; seems to be a shapely issue
E       AssertionError: assert 'POLYGON ((-6...4 -4, -4 -8))' == 'POLYGON ((6 ...-4 -8, 4 -8))'
E         - POLYGON ((6 10, 6 -10, -6 -10, -6 10, 6 10), (-4 4, 4 4, 4 8, -4 8, -4 4), (4 -8, 4 -4, -4 -4, -4 -8, 4 -8))
E         ?                                  ------                               -------                    ------
E         + POLYGON ((-6 10, 6 10, 6 -10, -6 -10, -6 10), (-4 8, -4 4, 4 4, 4 8, -4 8), (-4 -8, 4 -8, 4 -4, -4 -4, -4 -8))
E         ?           +       ++++++                          ++++++                 +  +        ++++++

sectionproperties/tests/test_sections.py:82: AssertionError
_____________________________________________________ test_nested_compound_geometry_from_points ______________________________________________________

    def test_nested_compound_geometry_from_points():
        """
        Tests a nested compound geometry can be built .from_points, that the control_points
        and hole nodes persist in the right locations, and that ...
        """
        points = [
            [-50.0, 50.0],
            [50.0, 50.0],
            [50.0, -50.0],
            [-50.0, -50.0],
            [37.5, -37.5],
            [37.5, 37.5],
            [-37.5, 37.5],
            [-37.5, -37.5],
            [25.0, -25.0],
            [25.0, 25.0],
            [-25.0, 25.0],
            [-25.0, -25.0],
            [12.5, -12.5],
            [12.5, 12.5],
            [-12.5, 12.5],
            [-12.5, -12.5],
        ]
        facets = [
            [0, 1],
            [1, 2],
            [2, 3],
            [3, 0],
            [4, 5],
            [5, 6],
            [6, 7],
            [7, 4],
            [8, 9],
            [9, 10],
            [10, 11],
            [11, 8],
            [12, 13],
            [13, 14],
            [14, 15],
            [15, 12],
        ]
        control_points = [[-43.75, 0.0], [-31.25, 0.0], [-18.75, 0.0]]
        holes = [[0, 0]]
        nested_compound = CompoundGeometry.from_points(
            points=points, facets=facets, control_points=control_points, holes=holes
        )
>       assert (
            nested_compound.geom.wkt
            == "MULTIPOLYGON (((50 50, 50 -50, -50 -50, -50 50, 50 50), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)), ((-37.5 -37.5, -37.5 37.5, 37.5 37.5, 37.5 -37.5, -37.5 -37.5), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)), ((-25 -25, -25 25, 25 25, 25 -25, -25 -25), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)))"
        )
E       AssertionError: assert 'MULTIPOLYGON...12.5 -12.5)))' == 'MULTIPOLYGON... 12.5 12.5)))'
E         - MULTIPOLYGON (((50 50, 50 -50, -50 -50, -50 50, 50 50), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)), ((-37.5 -37.5, -37.5 37.5, 37.5 37.5, 37.5 -37.5, -37.5 -37.5), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)), ((-25 -25, -25 25, 25 25, 25 -25, -25 -25), (12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5, 12.5 12.5)))
E         + MULTIPOLYGON (((-50 50, 50 50, 50 -50, -50 -50, -50 50), (12.5 -12.5, 12.5 12.5, -12.5 12.5, -12.5 -12.5, 12.5 -12.5)), ((37.5 -37.5, -37.5 -37.5, -37.5 37.5, 37.5 37.5, 37.5 -37.5), (12.5 -12.5, 12.5 12.5, -12.5 12.5, -12....
E         
E         ...Full output truncated (1 line hidden), use '-vv' to show

sectionproperties/tests/test_sections.py:176: AssertionError

```

The geometries appear to be identitcal however the point ordering differs - I see you've noted this in a comment but would be great to not have these fail for this arbitray reason?

**Comment by Spectre5** [comment]
For test code with dependent order, sorting is often needed for the comparisons.  Or conversion to a set.  But if sorting or converting, you may or may not need to do another check if the order matters!  Perhaps your can then grab this "minimum" from a sort and then reorder the list so that the "minimum" entry is first.

I run into this with paths and how they differ on Windows and Linux!

**Comment by connorferster** [comment]
Thanks @robbievanleeuwen and @Spectre5!

What branch is this on?

**Comment by robbievanleeuwen** [comment]
Yep was on v2 for me.

**Comment by connorferster** [comment]
Got it. In the process of merging/updating the `library...` branch. Will check that out when I finish.

**Comment by connorferster** [comment]
Fixed with #155
